geode/src/main/java/org/apache/calcite/adapter/geode/rel/GeodeSort.java

```diff
@@ -66,7 +66,7 @@ public class GeodeSort extends Sort implements GeodeRel {
 
   @Override public Sort copy(RelTraitSet traitSet, RelNode input,
       RelCollation newCollation, RexNode offset, RexNode fetch) {
-    return new GeodeSort(getCluster(), traitSet, input, collation, fetch);
```

Comment:
Use the new traitset in the copy of the sort, and the other sort copies as well.